### PR TITLE
Added Student Project by @duttaditya18 : Blog Engine made on Flask

### PR DESCRIPTION
**Pull request by nzec**
- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

* Added Student Project by @duttaditya18 : Blog Engine made on Flask
* Doesn't fixes any issue. (No other student project PR solved any issue.)

***Preview Link : https://duttaditya18.github.io/gci17.fossasia.org/#projects***

---
Ping @anshumanv @ParthS007

### CODE REVIEW

File: _data/student_projects.yml [hunk 1]
```diff
@@ -25,3 +25,8 @@
   github: https://atomickindness.tk/
   author: Ploypiti P.
   image: AAK_logo.png
+
+- name: Aditya Dutt
+  github: duttaditya18/beautiful-flask
```
Comment (Ploypiti) [inline _data/student_projects.yml]: Can you add `https://github/` before the link.

Comment (nzec) [reply _data/student_projects.yml]: Done!

**Review by Ploypiti — APPROVED** [review]
Nice job 👍

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by nikit19 — APPROVED** [review]
LGTM !

**Comment by nzec** [comment]
@nikit19 Thanks!

**Review by akshatnitd — APPROVED** [review]
LGTM ! Merging :rocket:

**Pull request merged**